geode-core/src/main/java/org/apache/geode/internal/i18n/LocalizedStrings.java

```diff
@@ -6236,7 +6236,7 @@
   public static final StringId RegionAttributesCreation__CLONING_ENABLE_IS_NOT_THE_SAME_THIS_0_OTHER_1 =
       new StringId(4741, "Cloning enabled is not the same: this:  {0}  other:  {1}");
   public static final StringId UpdateOperation_ERROR_APPLYING_DELTA_FOR_KEY_0_OF_REGION_1 =
-      new StringId(4742, "Error applying delta for key {0} of region {1}: {2}");
+      new StringId(4742, "Error applying delta for key {0} of region {1}");
```

Comment:
did you want to revert this change?

Reply:
Oh, yeah - thanks for noticing that Darrel.  I'll put those logging changes back in place when I merge to develop.  I don't think they warrant another precheckin run.